### PR TITLE
Fix #11456 - Check for update button

### LINKED ISSUE #11456: [MU4 Issue] Preferences > General > Languages - Check for Update button
**Describe the bug**
The check for update button in Preferences > Language is a bit confusing.

The placement of the button suggests that the action is connected to submitting the language selection.

**To Reproduce**
1. Go to Preferences > General > Languages
2. Select language from dropdown
3.Click 'Check for updates'
4. See issue

**Expected behavior**
As the check for updates function is logically separate from the language selection action, it could be clearer if there is more visual distinction.

Maybe this can be moved to the Updates section of Preferences?

**Screenshots**
Screen Shot 2022-04-29 at 12 12 05 PM

**Platform information**
OS: macOS 12.2
Arch.: x86_64
MuseScore version (64-bit): 4.0.0-2243301402
revision: github-musescore-musescore-a6ab311

### PR DESCRIPTION
**Pull request by lucnguyen93**
Resolves: https://github.com/musescore/MuseScore/issues/11456
"Check for update" button renamed to "Check for language updates" to avoid confusion with app update.
Moved button location to below the drop down for further clarity.

- [ x] I signed [CLA](https://musescore.org/en/cla)
- [ ]x I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [ x] I made sure the code compiles on my machine
- [ x] I made sure there are no unnecessary changes in the code
- [ x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [ x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [ x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/appshell/qml/Preferences/internal/LanguagesSection.qml [hunk 1]
```diff
@@ -66,23 +66,25 @@ BaseSection {
 
             onActivated: function(index, value) {
                 root.languageSelected(value)
+                }
             }
-        }
+         }
```
Comment (Jojo-Schmitz) [inline src/appshell/qml/Preferences/internal/LanguagesSection.qml:71]: whitespace damage

File: src/appshell/qml/Preferences/internal/LanguagesSection.qml [hunk 2]
```diff
@@ -66,23 +66,25 @@ BaseSection {
...
 
-        ProgressButton {
+     Row {
+         ProgressButton {
             id: progressBtn
 
             anchors.verticalCenter: parent.verticalCenter
 
-            text: qsTrc("appshell", "Check for update")
+            text: qsTrc("appshell", "Check for language updates")
```
Comment (Jojo-Schmitz) [inline src/appshell/qml/Preferences/internal/LanguagesSection.qml:79]: Wouldn#t that rather be "translations updates"?

**Comment by Jojo-Schmitz** [comment]
See also #11475

**Comment by RomanPudashkin** [comment]
https://github.com/musescore/MuseScore/pull/11475 was created before your PR, so I will close yours

**Pull request closed without merging**